ui/dashboards/src/components/Variables/Variable.tsx

```diff
@@ -281,7 +280,6 @@ function TextVariable({ name, source }: VariableProps) {
     <TextField
       title={tempValue as string}
       value={tempValue}
-      //onChange={(e) => setTempValue(e.target.value)}
```

Comment:
Comment ?

Reply:
Removing commented code, there is an other onChange just under this line